### DISCUSSION TITLE
Running a MAS using a .mas2j file without the MASConsole?

### DISCUSSION THREAD

**Discussion by mindcrime** [jason-lang/jason#107] [Q&A] [answered]
Hi all, am I right in thinking that currently there is no way to run a MAS using "jason some_mas_file.mas2j" without starting the MASConsole GUI? I've looked through the RunLocalMAS class and it seems to have calls to MASConsoleGUI.get() sprinkled in with no conditional logic so I'm thinking the only way to accomplish this would be to write a new class that serves the role of RunLocalMAS, but without the GUI references? Unless there is already a class in the code-base that does that, but at first blush I didn't see one.

**Comment by jomifred** [comment, marked as answer]
Hi Phillip, great that you started to code!

the Jason output is managed by a logging system (provided by Java). So to change from MASConsoleGUI to Console, you just need to edit the logging.properties file commenting the line where MASConsoleLogHandler is used and uncommenting the line that uses ConsoleHandler.

There is some doc at https://jason-lang.github.io/doc/faq.html#_how_to_log_the_overall_execution

**Reply by mindcrime** [reply]
Thank you sir! That worked like a charm.
